### ISSUE TITLE
Any chance we could have higher bitrate?

### ISSUE THREAD

**Issue by realdannys** [qrpike/spotify-playlist-downloader#27]
Be nice to use Apple Lossless (ALAC) so that we don't need to re-encode the files again. Plus they're 320k OGG as default to at least 320k AAC would be better.

**Comment by mikesmiffy128** [comment]
Hi, I decided to do a quick Google; apparently desktop Spotify uses Vorbis
while web Spotify uses MP3 (maybe for compatibility or something?). So
there's no way to do this except maybe libspotify (but that's deprecated,
probably to prevent tools like this :P), or by reverse engineering the
desktop protocol (which might be an interesting thing to try and do,
actually). Honestly though, a 320kbps MP3 is fine by me. 320kbps Vorbis is
maybe _unnoticably_ better, and of course I'd still rather stream at the
highest quality I can - but here we're talking about _ripping music_. If I
wanted or needed a perfectly lossless recording I'd probably just buy a CD.

but i mean whatever that's just how i feel about things

## 

Sent from Gmail Mobile

**Comment by realdannys** [comment]
Yeah of course, it uses MP3 as OGG isn't supported by most browsers (though ironically I don't believe Firefox supports MP3...I did setup a stream site once a few years ago)

I'd be happy with 320k mp3 sure - I mean...do we need to re-encode them at all? Seen as we're directly ripping the Mp3's here - or is that part of the decryption? In which case surely we can re-encode to any format via NODEJS the way they're being encoded now. Its just a case of adding more FFMPEG options....

**Comment by realdannys** [comment]
(of course I might be overlooking the fact that 160kbps IS the max bitrate for the web player)

**Comment by mikesmiffy128** [comment]
I'm pretty sure the default is a direct rip but I'll need check once I log
into my computer. If it is then you can just FFMpeg it from there I guess.
I imagine there's no point making it lossless though since the quality
isn't gonna get any higher (the only exception is if you're editing
something and you re-export, of course).

## 

Sent from Gmail Mobile

**Comment by realdannys** [comment]
If its direct rip then thats fine. Its only if a re-encode is required lossless would be helpful and only then to stop you encoding to MP3 twice (not to improve the quality of the existing mp3 obviously)

**Comment by shakisha** [comment]
any possibility to have got a better bitrate using this tool?
